requirements.txt

```diff
@@ -3,6 +3,6 @@ Faker==4.18.0
 tornado==6.0.4
 PyYAML==5.3.1
 jsonschema==3.2.0
-pybars4>=0.9.8
+pybars4 @ git+https://github.com/up9inc/pybars4@3f294fc32e9c9034a73022cf37c7cee070338b04
```

Comment:
PyPi does now allow this, it makes mockintosh not installable

Reply:
This is for demonstrating the current state and making the tests pass. I was going to change it to something like `pybars4>=0.9.9` in the end.